src/moonlink_connectors/src/pg_replicate/table_init.rs

```diff
@@ -70,6 +71,7 @@ pub async fn build_table_components(
         }),
     );
 
+    let wal_config = WalConfig::default_wal_config_local(table_id, &PathBuf::from(base_path));
```

Comment:
for the directory you use, you need to make sure it exists

Reply:
I think you should handle it in this PR, otherwise pg_mooncake could potentially crash